code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -251,10 +252,13 @@ obj/structure/bed/Destroy()
 	if(istype(R,/obj/structure/bed/medevac_stretcher)) //We need to preserve key variables like linked beacons and cooldowns.
 		var/obj/item/roller/medevac/I = src
 		var/obj/structure/bed/medevac_stretcher/B = R
-		B.last_teleport = I.last_teleport
-		B.linked_beacon = I.linked_beacon
-		if(B.linked_beacon.linked_bed == src)
-			B.linked_beacon.linked_bed_deployed = B
+		if(last_teleport)
```

Comment:
Why are you passing a `last_teleport` var in the call of this proc that's the same name as the variable defined at the`/obj/item/roller/medevac` level?
It does work, but it's confusing.
Do you even need to to pass that argument when you can just do `if(I.last_teleport)` instead?